src/Jwt/Udb3Token.php

```diff
@@ -34,6 +34,9 @@ public function id(): string
             return $this->token->getClaim('https://publiq.be/uitidv1id');
         }
 
+        if ($this->endsWith($this->token->getClaim('sub'), self::CLIENTS)) {
+            return 'client|' . rtrim($this->token->getClaim('sub'), self::CLIENTS);
+        }
```

Comment:
Like I said in Slack I would keep the suffix and not prefix it, to avoid confusion in the future. The goal of the prefix was just to have a way to quickly see if a "user" id was a client or not, and we can do that with the `@clients` suffix from Auth0 which we just didn't see at first. If we change it to a custom prefix it will probably get confusing in the long run because it's then different from what is used in Auth0 itself.